Tasks.md

```diff
@@ -29,7 +29,7 @@ On GitKraken, right click on the latest `main` branch commit, and select create
 
 6. Stage and commit your changes to that file. 
 
-7. Push your commits to `main`. This saves your commits to a branch on the remote repo, which is in this case on GitHub.
+7. Push your commits to your repo. This saves your commits to a branch on the remote repo, which is in this case on GitHub.
```

Comment:
While I applaud you trying to fix @KipHamiltons horrendous typos, this needs to be done in a separate PR.

Reply:
Liam and I fixed this up in PR #15